Description: Point the online help to /usr/share/identicurse/README.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=612976
Author: Alessio Treglia <alessio@debian.org>

--- a/src/identicurse/tabbage.py
+++ b/src/identicurse/tabbage.py
@@ -225,6 +225,8 @@ class Help(Tab):
     def __init__(self, window, identicurse_path):
         self.name = "Help"
         self.path = os.path.join(identicurse_path, "README")
+        if not os.path.isfile(self.path):
+            self.path = '/usr/share/identicurse/README'
         Tab.__init__(self, window) 
 
     def update(self):
